Tests/PointFreeTests/AccountTests/__Snapshots__/AccountTests/testAccountWithDiscount.1.Conn.txt

```diff
@@ -1875,7 +1875,7 @@ html {
                         <div class=" col-m col-d-6 col-m-12">
                           <div class="m-pl1 m-pr1 end-d">
                             <p>
-                              <a class="medium cursor-pointer nowrap btn-normal rounded border-none text-decoration-none pf-link-white fg-white bg-purple h6 m-pl1 m-pr1 m-pt1 m-pb1" href="/account/payment-info">
+                              <a class="medium cursor-pointer nowrap btn-normal rounded border-none text-decoration-none pf-link-white fg-white bg-purple h6 m-pl1 m-pr1 m-pt1 m-pb1" href="/account/payment-info?">
```

Comment:
oh this is an interesting artifact. guess it's totally ok?

Reply:
I'm fixing in swift-web.